CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### CHANGE LOG
 
+### v.2.4.10
+> Fixes outstanding HTML conformance issues
+  - Fixes CSS: border-radius: Too many values or values are not recognized
+
```

Comment:
@ktp242 Like this one for CHANGELOG?

Reply:
I think you can add the change to the already existing v2.4.10